libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/model/TextEditorState.kt

```diff
@@ -45,6 +45,20 @@ sealed interface TextEditorState {
         is Rich -> richTextEditorState.hasFocus
     }
 
+    suspend fun setHtml(html: String) {
+        when (this) {
+            is Markdown -> Unit
+            is Rich -> richTextEditorState.setHtml(html)
+        }
+    }
+
+    suspend fun setMarkdown(text: String) {
+        when (this) {
+            is Markdown -> state.text.update(text, true)
+            is Rich -> richTextEditorState.setMarkdown(text)
+        }
+    }
+
```

Comment:
Isn't this already included in the latest `develop`? Maybe the PR needs to merge the latest changes?